js-wether/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="jp">
+
+<head>
+    <meta charset="UTF-8">
+    <title>JSwether</title>
+</head>
+
+<body>
+    <div>
+        <h2>世界の天気</h2>
+    </div>
+    <div class="location">
+        <select id='city'>
```

Comment:
こちらだけシングルクォーテーションとなっています。
どちらを利用しても良いのですが、1プロジェクト内では統一しましょう

```suggestion
        <select id="city">
```

補足：
現場の話です。こういったコーディングスタイルは、会社全体というよりは、プロジェクト内で統一を頑張ることが多いです。